src/Files.App/Helpers/Win32/Win32Helper.Storage.cs

```diff
@@ -935,6 +937,40 @@ public static bool UnsetFileAttribute(string lpFileName, FileAttributes dwAttrs)
 			return Win32PInvoke.SetFileAttributesFromApp(lpFileName, lpFileInfo.dwFileAttributes & ~dwAttrs);
 		}
 
+		public static unsafe bool SetCompressionAttributeIoctl(string lpFileName, bool isCompressed)
+		{
+			using var hFile = PInvoke.CreateFile(
+				lpFileName,
+				Win32PInvoke.GENERIC_READ | Win32PInvoke.GENERIC_WRITE,
```

Comment:
```suggestion
				(uint)FILE_ACCESS_RIGHTS.FILE_WRITE_ATTRIBUES,
```

Reply:
If it doesn't work well, let me know. I think standard write access works as well.